### PR TITLE
Bugfix for bugfix for printing of affine schemes

### PR DESCRIPTION
**Pull request by thofma**
Turns
```
julia> [Spec(L)]
1-element Vector{AffineScheme{QQField, Oscar.MPolyLocRing{QQField, QQFieldElem, QQMPolyRing, QQMPolyRingElem, Oscar.MPolyPowersOfElement{QQField, QQFieldElem, QQMPolyRing, QQMPolyRingElem}}}}:
AA^2 \ scheme((x)*(y)*(x*y))
```
into
```
julia> [Spec(L)]
1-element Vector{AffineScheme{QQField, Oscar.MPolyLocRing{QQField, QQFieldElem, QQMPolyRing, QQMPolyRingElem, Oscar.MPolyPowersOfElement{QQField, QQFieldElem, QQMPolyRing, QQMPolyRingElem}}}}:
AA^2 \ scheme(x*y*x*y)
```

I dedicate this to Dan S., who taught us to avoid parentheses like never before.

### CODE REVIEW

**Review by HechtiDerLachs — APPROVED** [review]
Thanks! 

I have little idea of what's really happening in the background, but I am leaning towards trusting you and @tthsqe12 on their good work. With the missing bracket added, maybe we do not even need to fix the doctests.

Edit: It seems that I don't have the right to push to this pr. I will leave a suggestion, then.

File: src/AlgebraicGeometry/Schemes/AffineSchemes/Objects/Methods.jl [hunk 1]
```diff
@@ -78,12 +78,18 @@ function _show(io::IO, X::AbsAffineScheme{<:Any, <:MPolyQuoLocRing{<:Any, <:Any,
...
     print(io, raw")")
-  elseif isone(length(denominators(S)))
-    print(io, raw") \ scheme(", first(denominators(S)), ")")
   else
-    print(io, raw") \ scheme((")
-    join(io, denominators(S), ")*(")
-    print(io, "))")
+    AA = AbstractAlgebra
+    PP = AbstractAlgebra.PrettyPrinting
+    print(io, raw" \ ")
```
Comment (HechtiDerLachs) [inline src/AlgebraicGeometry/Schemes/AffineSchemes/Objects/Methods.jl]:
```suggestion
    print(io, raw") \ ")
```

**Review by fingolfin — APPROVED** [review]
Very nice. Can/should this be backported? (I don't know if it affects the book)

**Comment by HechtiDerLachs** [comment]
Yes, I think it should be backported. At least #3437 must be backported, since the output is just wrong. Then this one doesn't seem to hurt on top of it.

**Pull request merged**